solidstart-tanstackquery-tailwind-modules/src/types/user-profile-type.ts

```diff
@@ -1,21 +1,21 @@
-export interface UserProfile {
+export interface UserProfileProps {
```

Comment:
As noted above, this change does not seem like a good idea.

I would keep the component props type separate, and maybe even be a subset or extension of this `UserProfile` type, but don't put the props type here

Reply:
So this type also matches the API result we get from the query, and fits precisely with the props the component requires, that this why it is declared in the types/ to be used in places it is needed.